components/SiteCard.tsx

```diff
@@ -32,37 +34,60 @@ const SiteCard = React.forwardRef<HTMLAnchorElement, SiteCardProps>(
           }`}
         >
           <div className="flex flex-col flex-1 w-[60%] p-4 pl-5">
-            <h2 className="text-amber text-xl font-semibold">
-              {isLoading ? 'Loading...' : client_site_name ?? 'My Site'}
-            </h2>
-            <div className="flex flex-col mt-2 gap-1">
-              <p className="text-ocf-gray-500 text-xs font-medium">
-                {`Current output: ${
-                  currentOutput != undefined
-                    ? currentOutput.toFixed(2) + ' kW'
-                    : 'loading...'
+            <div className="flex flex-col flex-1 p-4 pl-5">
+              <h2
+                className={`text-amber text-xl font-semibold ${
+                  isLoading || currentOutput == undefined ? skeleton : ``
```

Comment:
When would `currentOutput` be `undefined`? Wondering if a skeleton would actually be the correct thing to show in this specific case...